_codux/boards/services-section.board.tsx

```diff
@@ -0,0 +1,74 @@
+import { createBoard } from '@wixc3/react-board';
+import classNames from 'classnames';
+import styles from './services-section.board.module.scss';
+import commonStylesStyles from '../../src/styles/common-styles.module.scss';
+import LightningSvg from '../../src/assets/lightning.svg';
+import AnchorSvg from '../../src/assets/anchor.svg';
+import LeafSvg from '../../src/assets/leaf.svg';
+import GlobeSvg from '../../src/assets/globe.svg';
+
+export default createBoard({
+    name: 'ServicesSection',
+    Board: () => (
+        <div className={styles.servicesSection}>
+            <div className={styles.content}>
+                <h2 className={styles.servicesTitle}> Write a Title Here</h2>
+                <p className={styles.servicesDescription}>
+                    This is the space to introduce visitors to the business or brand. Briefly
+                    explain who&apos;s behind it, what it does and what makes it unique. Share its
+                    core values and what this site has to offer.
+                </p>
+                <button className={commonStylesStyles.secondaryButtonWidth}>Explore</button>
```

Comment:
@ynesteruk do our built-in board snippets support multiple CSS files in a single snippet?

@RanBerkovich1 in any case don't name it `commonStylesStyles`. And `secondaryButtonWidth` doesn't make sense as a common style.

Reply:
Also I see this project imports `src/styles/index.scss` into every board via `board-global-setup`. Any styles coming from `index.css` will not be available in the snippet.

E.g. the this font will not be used:
https://github.com/wixplosives/wireframe-snippets/blob/32b5f322a0f7aae58c3a0c1331c60a9ec737f188/src/styles/index.scss#L5-L14

I don't think you can currently add a custom font to the snippet. You'll either have to omit the font entirely, and then it will depend on font used in each specific project. Or you can hardcode some built-in font like Helvetica, but then users will need to replace it each time after adding a snippet.

Reply:
> do our built-in board snippets support multiple CSS files in a single snippet?

No. at the moment it should be only one plain css file that located in the same folder and named according to our convetion (e.g. `example.board.css` belongs to `example.board.tsx`)

Reply:
@ynesteruk so no scss files, no css modules, no scoping?